runtime/testy/runtime.py

```diff
@@ -322,9 +322,18 @@ def startHibike(badThingsQueue, stateQueue, pipe):
   # badThingsQueue - queue to runtime
   # stateQueue - queue to stateManager
   # pipe - pipe from statemanager
+  def addPaths():
+    """Modify sys.path so we can find hibike.
+    """
+    path = os.path.dirname(os.path.abspath(__file__))
+    parent_path = path.rstrip("runtime/testy")
```

Comment:
Your call, but maybe `os.path.join(path, '../../hibike')` instead of this line and the next?